spv/envelope.go

```diff
@@ -9,13 +12,14 @@ import (
 
 // Envelope is a struct which contains all information needed for a transaction to be verified.
 //
-// spec at https://tsc.bitcoinassociation.net/standards/spv-envelope/
 type Envelope struct {
-	TxID          string               `json:"txid,omitempty"`
-	RawTx         string               `json:"rawTx,omitempty"`
-	Proof         *bc.MerkleProof      `json:"proof,omitempty"`
-	MapiResponses []bc.MapiCallback    `json:"mapiResponses,omitempty"`
-	Parents       map[string]*Envelope `json:"parents,omitempty"`
+	TxID          string
+	RawTx         string
+	Proof         *bc.MerkleProof
+	MapiResponses []bc.MapiCallback
+	Parents       map[string]*Envelope
+	Transactions  map[string]*Envelope
```

Comment:
what are the newly added `Transactions` and `Verified` fields for?

Reply:
Well spotted - I think they were added while we were discussing hypotheticals on a call together, used as an example. Not used in actual code anywhere. oops!